### PR TITLE
For #14922: Make add custom search engine more discoverable

### LINKED ISSUE #14922: Users are unable to find "add custom search engine"
## User Story
- As a user, I want to know that I can add a custom search engine. 

## User problem
- According to user reviews, users don't know that this feature exists. This feature may be hard to find on a very busy Search settings page and making it stand out more may address this issue. 

### Acceptance Criteria
- I can quickly find the "add custom search engine" feature.

### PR DESCRIPTION
**Pull request by mcarare**
Fixes #14922 

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Review by apbitner — APPROVED** [review]

**Comment by gabrielluong** [comment]
We might want to consider using padding instead of margin so that the ripple will expand through the entire row.

Screen Shot 2020-10-15 at 2 26 25 PM
Screen Shot 2020-10-15 at 2 26 14 PM

File: app/src/main/res/layout/preference_search_add_engine.xml [hunk 1]
```diff
@@ -10,14 +10,14 @@
     android:layout_width="match_parent"
     android:background="?android:selectableItemBackground"
     android:clickable="true"
-    android:focusable="true">
+    android:focusable="true"
+    android:layout_marginStart="@dimen/top_bar_alignment_margin_start">
```
Comment (gabrielluong) [inline app/src/main/res/layout/preference_search_add_engine.xml:14]: If we switch from margin to padding, the ripple background should be expand the full width of the row.

```suggestion
    android:paddingStart="@dimen/top_bar_alignment_margin_start"
    tools:ignore="RtlSymmetry">
```

**Review by gabrielluong — APPROVED** [review]
Changes look good. I think we just want to address the ripple background in the review comments. r+ assuming the changes are made.

**Comment by gabrielluong** [comment]
Closing this for https://github.com/mozilla-mobile/fenix/pull/15935 since I couldn't push the suggestion here.

**Pull request closed without merging**